readme.md

```diff
@@ -67,7 +67,7 @@ Type: `Array`
 Specify components to be excluded from being added to the RequireJS config.
 
 ```js
-bower: {
+'bower-requirejs': {
```

Comment:
hehe,... 
i somehow expected this comment ;)
But i'm fine with `bowerRequirejs`.
Thanks for the fast review